### DISCUSSION TITLE
Wrong dependency

### DISCUSSION THREAD

**Discussion by SteveDunn** [community/community#14362] [Code Security] [answered]
I'm seeing this dependency in [Vogen](https://github.com/SteveDunn/Vogen/network/dependencies#.github%252Fworkflows%252Fcodeql-analysis.yml)

![image](https://user-images.githubusercontent.com/263416/161906656-4d18c8fe-f8c9-41e8-8c68-b334b6114dc2.png)

I think it's wrong. It shows as this in the project itself:

![image](https://user-images.githubusercontent.com/263416/161906737-086133a3-0aff-42aa-b240-eed889c48304.png)

Any ideas?

**Comment by jhutchings1** [comment, marked as answer]
@SteveDunn Thanks for reporting this. Looks like our package to repository mapping is bad. I'll direct this to engineering to have a look.
